src/Core/Geographic/Extent.js

```diff
@@ -69,6 +88,8 @@ class Extent {
      * Extent is geographical bounding rectangle defined by 4 limits: west, east, south and north.
      * If crs is tiled projection (WMTS or TMS), the extent is defined by zoom, row and column.
      *
+     * Warning, using geocentric projection isn't make with geographical extent.
```

Comment:
is it really "isn't make" or rather "isn't consistent"?

Reply:
or "isn't possible"?

Reply:
`isn't consistent`

I change